packages/main/test/specs/Select.spec.js

```diff
@@ -615,6 +615,21 @@ describe("Select general interaction", () => {
 		const selectedOption = await popover.$("ui5-list").$("ui5-li[selected]");
 		assert.ok(await selectedOption.isClickable(), "Selected option is visible in the viewport.");
 	});
+
+	it("tests tooltip property of ui5-option", async () => {
+		const firstOption = await browser.$("#selectTooltip ui5-option:first-child");
+		const initialValue = "Cozy";
+		let tooltipValue = await firstOption.getProperty("tooltip");
+
```

Comment:
We can only test if the tooltip is correctly propagated. No need to check framework side if it's changed due property change

Reply:
I meant to check if the tooltip is propagated to the list item correctly. Here is an example (in addition to make it work we have to add `stable-dom-ref="tooltip-test-option"` attribute to the option that we are testing):
```js
	it("Tooltip propagation", async () => {
		const select = await browser.$("#mySelect");
		const tooltip = "Tooltip text"

		await select.click();

		const popover = await select.shadow$("[ui5-responsive-popover]");
		const option = await select.shadow$("[ui5-option]");
		const listItem = await popover.$("[data-ui5-stable='tooltip-test-option']")

		assert.strictEqual(await option.getProperty("tooltip"), tooltip, "Tooltip is correct");
		assert.strictEqual(await listItem.getProperty("tooltip"), tooltip, "Tooltip is correctly propagated");
	});
```